honeybee_radiance/postprocess/en17037.py

```diff
@@ -0,0 +1,264 @@
+"""Functions for post-processing EN 17037 daylight outputs."""
+import json
+import os
+
+from .annual import filter_schedule_by_hours, _process_input_folder
+
+
+def _metrics(values, occ_pattern, threshold, total_hours):
```

Comment:
This name doesn't seem quite correct. There's only one metric, right? Maybe we should just call it `_daylight_autonomy`